lib/auth/apiserver.go

```diff
@@ -140,8 +140,11 @@ func NewAPIServer(config *APIConfig) http.Handler {
 	srv.DELETE("/:version/tunnelconnections/:cluster/:conn", srv.withAuth(srv.deleteTunnelConnection))
 	srv.DELETE("/:version/tunnelconnections/:cluster", srv.withAuth(srv.deleteTunnelConnections))
 	srv.DELETE("/:version/tunnelconnections", srv.withAuth(srv.deleteAllTunnelConnections))
+	// TODO(awly): migrate these to the gRPC service.
 	srv.POST("/:version/kube_services", srv.withAuth(srv.upsertKubeService))
 	srv.GET("/:version/kube_services", srv.withAuth(srv.getKubeServices))
+	srv.DELETE("/:version/kube_services/:name", srv.withAuth(srv.deleteKubeService))
+	srv.DELETE("/:version/kube_services", srv.withAuth(srv.deleteAllKubeServices))
```

Comment:
If these are new, why not start them off as gRCP methods?

Reply:
I wrote these before realizing my mistake :)
Rewrite to gRPC should happen before the release anyway, I'd rather get this PR in as is and make a follow-up